Null pointer dereference in MainActivity.getApplicationName()

**Describe the bug**
The following NPE is thrown:
```
     Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'java.lang.String java.lang.CharSequence.toString()' on a null object reference
        at com.chuckerteam.chucker.api.internal.ui.MainActivity.getApplicationName(MainActivity.java:90)
        at com.chuckerteam.chucker.api.internal.ui.MainActivity.onCreate(MainActivity.java:46)
        at android.app.Activity.performCreate(Activity.java:7802)
```
Affected line:
https://github.com/ChuckerTeam/chucker/blob/c95f7f19f1df188af1e0fc53245dcc50132c1004/library/src/main/java/com/chuckerteam/chucker/api/internal/ui/MainActivity.java#L90
invokes `toString()` on `ApplicationInfo.nonLocalizedLabel` which may me null.
Official documentation on https://developer.android.com/reference/android/content/pm/PackageItemInfo.html#nonLocalizedLabel says

> The string provided in the AndroidManifest file, if any. 

If there is no such string in manifest then value is `null`.

**To Reproduce**
1. Invoke `context.startActivity(Chucker.getLaunchIntent(context, Chucker.SCREEN_HTTP))` where `context` is valid activity `Context`.

Sample project:
https://github.com/DroidsOnRoids/FoQA
Use `Chucker` Hyperion menu option to reproduce this issue.

**Expected behavior**
There should be no crash.

**Smartphone**
 - Device: does not matter but e.g. emulator
 - OS: does not matter but e.g. 5.0
 - Lib version: 3.0.0-alpha2

--- a/library/src/main/java/com/chuckerteam/chucker/api/internal/ui/MainActivity.java
+++ b/library/src/main/java/com/chuckerteam/chucker/api/internal/ui/MainActivity.java
@@ -84,10 +84,9 @@ public class MainActivity extends BaseChuckerActivity implements TransactionAdap
         }
     }
 
-    private String getApplicationName() {
+    private CharSequence getApplicationName() {
         ApplicationInfo applicationInfo = getApplicationInfo();
-        int stringId = applicationInfo.labelRes;
-        return stringId == 0 ? applicationInfo.nonLocalizedLabel.toString() : getString(stringId);
+        return applicationInfo.loadLabel(getPackageManager());
     }
 
     @Override

